extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -652,12 +665,20 @@ export class AmpA4A extends AMP.BaseElement {
           });
         }))
         // some() returns an array of which we only need a single value.
-        .then(returnedArray => returnedArray[0]);
+        .then(returnedArray => returnedArray[0], () => {
```

Comment:
Style debate:  Should we prefer `.then(onFulfilled, onRejected)` style or `.then(onFullfilled).catch(onRejected)` style?  I feel the latter is slightly more readable, but I'm flexible.  I haven't surveyed the AMP code base to see if there's a clear preference.